domains/raiton.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "RaitonRed",
+        "email": "alirzahosinopoor.1390@gmail.com"
+    },
+    "records": {
+        "TXT": "vc-domain-verify=raiton.is-a.dev,1f4f7e14825ff4c6cca2",
+        "A": "216.198.79.1"
```

A records is an Array
```suggestion
        "A": ["216.198.79.1"]
```